src/sound/jack/sound.h

```diff
@@ -34,15 +34,15 @@
 #include <stdio.h>
 #include <QThread>
 #include <string.h>
-#include "../../util.h"
-#include "../soundbase.h"
-#include "../../global.h"
-
 #if WITH_JACK
 #    include <jack/jack.h>
 #    include <jack/midiport.h>
 #endif
 
+#include "../../global.h"
+#include "../../util.h"
+#include "../soundbase.h"
+
```

Comment:
Windows/Jack build reproducibly [fails](https://github.com/jamulussoftware/jamulus/runs/6544938563?check_suite_focus=true#step:9:278) due to some definition problem. Maybe this include reordering is the culprit?

Reply:
`"../soundbase.h"` includes `"../../global.h"` and `"../../util.h"` (and in that order) -- this _should_ work.  Also, I can't see _any_ jack-ish dependency...  It's really weird but yeah, totally reproducible.  I'll try dropping the includes here...  Mm... better.

Reply:
Hm.  And then it didn't.  I've given in and reverted to just the needed change in the sound header, which seems to make it work.  It would be good to know why the reordering is fine on Linux but not Windows.